library/src/ui/index.js

```diff
@@ -85,4 +88,162 @@ window.CoNDeT.ui = {
       this.removeChild(childToRemove);
     },
   },
+
+  Table: (function () {
+    function constructor() {}
+
+    constructor.prototype = Object.create(window.CoNDeT.ui.BaseComponent);
+
+    constructor.prototype.onInit = function (common, props) {
+      this.ref = document.createElement("table");
+      this.ref.classList.add("condet-table");
+      common.deltaXY = this.getPosition();
+      this.ref.style.cssText = `position: absolute; left: ${common.deltaXY.x}; top: ${common.deltaXY.y}`;
```

Comment:
Nope. Tutaj nadpisujemy w ten sposób przekazany obiekt. (Obiekty w JS są mutable). Należy zrobić zmienną która sumuje `x` z `common.deltaXY` wraz z `x` z `this.getPosition()` (analogicznie y) i dopiero użyć do ustawienia position aboslute. Nie wiem czy nie przeniesiemy się na transform translate, ale to nie w tym sprincie